mentat/conversation.py

```diff
@@ -145,17 +145,21 @@ async def get_model_response(self) -> list[FileEdit]:
         stream = SESSION_STREAM.get()
         code_context = CODE_CONTEXT.get()
         cost_tracker = COST_TRACKER.get()
-        code_file_manager = CODE_FILE_MANAGER.get()
         parser = PARSER.get()
 
         messages = self.messages.copy()
 
-        code_file_manager.read_all_file_lines()
+        # Rebuild code context with active code and available tokens
+        tokens = await num_tokens_from_messages(messages, self.model)
```

Comment:
this new function `num_tokens_from_messages`, only supports gpt 3.5 and 4 models - can we use the count_tokens function we use in other places? This one is probably more exact, I'm not sure

Reply:
Sure - so I'll get rid of num_tokens_from_messages, add a comment with the link for future reference, and set Conversation back to the way it was before.